src/Microsoft.Net.Http.Headers/ContentDispositionHeaderValue.cs

```diff
@@ -462,6 +462,16 @@ private bool IsQuoted(string value)
                 && value.EndsWith("\"", StringComparison.Ordinal);
         }
 
+        /// <summary>
+        /// Unquotes string if it is not null. Does nothing if the string is null.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        private string Unquote(string value)
```

Comment:
Or maybe even just make the changed, and commit usint `git commit --amend` to amend the last commit. Then you can force push to this PR.